core/platform/auth/firebase_auth_services.py

```diff
@@ -102,6 +102,7 @@ def establish_auth_session(request, response):
         fresh_cookie = firebase_auth.create_session_cookie(
             token, feconf.FIREBASE_SESSION_COOKIE_MAX_AGE)
 
+    # Check that the create_session_cookie function has not failed.
```

Comment:
This raises the question: what should happen if it has? Presumably not just "carry on as if nothing had happened..."?

I think the comment should be more about explaining what cases can cause fresh_cookie to be None.

Reply:
Obsolete since we're just letting the exceptions raise.